### PR TITLE
(DOCSP-12343): Specify $function operator support

### PR DESCRIPTION
**Pull request by cbush**
- Adds section to crud-and-aggregation-apis for aggregation operator support exceptions

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-12343

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/function-agg-op/mongodb/crud-and-aggregation-apis.html#aggregation-pipeline-operator-availability

### CODE REVIEW

**Review by kpatel71716 — APPROVED** [review]
LGTM!

File: source/mongodb/crud-and-aggregation-apis.txt [hunk 1]
```diff
@@ -520,22 +520,22 @@ Query Option Availability
 Aggregation
 -----------
 
+Aggregation Pipeline Stage Availability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```
Comment (nathan-contino-mongo) [inline source/mongodb/crud-and-aggregation-apis.txt:525]: Correct me if I'm wrong, but I don't think I've ever seen a header with a subheader immediately following. It looks like the other sections on this page usually put the subheader right before the table, after the description paragraph. Could we do that here for consistency?

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM with one suggested change for consistency.

File: source/mongodb/crud-and-aggregation-apis.txt (hunk 1 above)
Comment (cbush) [reply source/mongodb/crud-and-aggregation-apis.txt:525]: I moved the description for stage availability, which was above the stage availability section, down to underneath that header and added the operator availability header with its own description. I can combine both descriptions up top and leave the subheaders without descriptions. Thoughts?

Comment (nathan-contino-mongo) [reply source/mongodb/crud-and-aggregation-apis.txt:525]: Ah, I see what you mean now. It's just that the aggregation section has an extra level of hierarchy compared to some of the other sections. This is fine then!

**Pull request merged**